### ISSUE TITLE
Files from `file_cache` are persisting

### ISSUE THREAD

**Issue by luiztauffer** [flatironinstitute/dendro-old#139]
The files stored at `file_cache` are persisting on apptainer mode:

![image](https://github.com/flatironinstitute/dendro/assets/24541631/abb34953-39e3-4bf9-a7d0-31a562ec1a07)

**Comment by magland** [comment]
I added the file cache in order to enable jobs to use the outputs of previous jobs without needing to re-download. It also enables pipelines where the intermediate files are not uploaded to the cloud.

I suppose we should make some configuration option to either not use this cache, or to automatically clear old files.

**Comment by luiztauffer** [comment]
yeah that makes sense, probably at the job config level, options like these?
- [ ] use cache from previous jobs
- [ ] clean cache from this job

![image](https://github.com/flatironinstitute/dendro/assets/24541631/2f9eb6aa-e3a9-4d41-aeb2-98053e026d79)
